lib/features/family/features/topup/repository/topup_repository.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:givt_app/features/family/network/api_service.dart';
+
+mixin TopupRepository {
+  Future<bool> topupChild(String childGUID, int amount);
+}
+
+class TopupRepositoryImpl with TopupRepository {
+  TopupRepositoryImpl(
+    this._apiService,
+  );
+
+  final FamilyAPIService _apiService;
+
+  @override
+  Future<bool> topupChild(String childGUID, int amount) async {
+    final response = await _apiService.topUpChild(childGUID, amount);
```

Comment:
Currently we have the EditChildRepository and the TopupRepository both doing a topup api call. The first one is doing the v1 call and the 2nd one is doing the v2 call. The first one exposes a stream that updates others that something regarding the child has changed. I don't think this one should expose a different stream because then we lose sight of where to get child data updates.

Do we still need the v1 call? Else i'd say we change the EditChildRepository to use the v2 call and then delete this repo.